examples/plugins/transform/pipeline_operator_plugin.cpp

```diff
@@ -6,20 +6,20 @@
 // SPDX-FileCopyrightText: (c) 2021 The VAST Contributors
 // SPDX-License-Identifier: BSD-3-Clause
 
```

Comment:
One additional note that I forgot: This needs a changelog entry in the `changes` category.

(Just commented on a meaningless place so this can actually be resolved)